instrumentation/sinatra/lib/opentelemetry/instrumentation/sinatra/middlewares/tracer_middleware.rb

```diff
@@ -15,35 +16,15 @@ def initialize(app)
           end
 
           def call(env)
-            extracted_context = OpenTelemetry.propagation.extract(
-              env,
-              getter: OpenTelemetry::Common::Propagation.rack_env_getter
-            )
-            OpenTelemetry::Context.with_current(extracted_context) do
-              tracer.in_span(
-                env['PATH_INFO'],
-                attributes: { 'http.method' => env['REQUEST_METHOD'],
-                              'http.url' => env['PATH_INFO'] },
-                kind: :server
-              ) do |span|
-                @app.call(env).tap { |resp| trace_response(span, env, resp) }
-              end
-            end
+            @app.call(env).tap { trace_response(env) }
```

Comment:
Will this bypass trace response if the request raises an error?

Reply:
good spot, I've made a change for this